snap/Makefile

```diff
@@ -19,6 +19,9 @@ clean:
 kubectl:
 	@KUBE_VERSION=${KUBE_VERSION} ${build} kubectl
 
+kubeadm:
```

Comment:
Add this to [targets](https://github.com/juju-solutions/release/blob/f2e8c73a009509205fb48f348bfa125832b877c7/snap/Makefile#L7), too?